Formula/goquiet.rb

```diff
@@ -0,0 +1,23 @@
+class Goquiet < Formula
+  desc "Shadowsocks plugin that obfuscates the traffic"
+  homepage "https://github.com/cbeuw/GoQuiet"
+  url "https://github.com/cbeuw/GoQuiet/archive/v1.0.2.tar.gz"
+  sha256 "bdd7644eb9af4130c5be461b55bb3f3eeb244e2ac2a172208f60e24f662c32c0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/cbeuw/GoQuiet").install buildpath.children
+
+    cd "src/github.com/cbeuw/GoQuiet" do
+      system "make"
+      bin.install Dir["build/*"]
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Reply:
Done, thank you for the review!